pom.xml

```
@@ -36,12 +36,12 @@
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <!-- Dependencies - Jakarta -->
-        <version.jakarta.api>8.0.0</version.jakarta.api>
-        <version.jakarta.annotation>1.3.5</version.jakarta.annotation>
-        <version.jakarta.cdi>2.0.2</version.jakarta.cdi>
-        <version.jakarta.ws-rs>2.1.6</version.jakarta.ws-rs>
-        <version.jakarta.jsonp>1.1.6</version.jakarta.jsonp>
-        <version.jakarta.jsonb>1.0.2</version.jakarta.jsonb>
+        <version.jakarta.api>9.0.0</version.jakarta.api>
+        <version.jakarta.annotation>2.0</version.jakarta.annotation>
```

```suggestion
        <version.jakarta.annotation>2.0.0</version.jakarta.annotation>
```
https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/2.0.0/jar

---

similarly for others, CDI should be 3.0.0, JAX-RS 3.0.0, etc.

---

basically, 2.0.0 is the same as 2.0. Anyway, I can update to reflect the link.

---

No, it isn't :) -
```
[ERROR] Failed to execute goal on project dependency-test: Could not resolve dependencies for project io.xstefank:dependency-test:jar:0.0.1: Could not find artifact jakarta.annotation:jakarta.annotation-api:jar:2.0 in central (https://repo.maven.apache.org/maven2)
```

---

ah, good point. They are the same from semantic versioning point of view :).